modules/sdk-core/src/bitgo/utils/tss/ecdsa/types.ts

```diff
@@ -1,19 +1,19 @@
 import { ECDSA } from '../../../../account-lib/mpc/tss';
 import { ECDSAMethodTypes } from '../../../tss/ecdsa';
-import { BackupKeyShare, CreateKeychainParamsBase } from '../baseTypes';
-import { Key, SerializedKeyPair } from 'openpgp';
+import { BackupKeyShare, CreateKeychainParamsBase, backupGpgKey } from '../baseTypes';
+import { Key } from 'openpgp';
 import { BackupProvider } from '../../../wallet';
 
 export type KeyShare = ECDSA.KeyShare;
 export type DecryptableNShare = ECDSAMethodTypes.DecryptableNShare;
 
 export type CreateEcdsaKeychainParams = CreateKeychainParamsBase & {
   userKeyShare: ECDSA.KeyShare;
-  backupGpgKey: SerializedKeyPair<string>;
   backupKeyShare: BackupKeyShare;
   isThirdPartyBackup?: boolean;
   backupProvider?: BackupProvider;
   bitgoPublicGpgKey: Key;
+  backupGpgKey: backupGpgKey;
```

Comment:
nit: can we make this consistent to call `backupPublicGpgKey`?

Reply:
it's not necessarily just the public, it can be either a key pair or just a pub key